lib/main.js

```diff
@@ -1,85 +1,64 @@
 'use strict';
 
-var _		= require('lodash');
-var request	= require('request');
-
-var dashboard = require('./oo_dashboard');
-var executions = require('./oo_executions');
-var schedules = require('./oo_schedules');
-var config = require('./oo_config');
-
-/**
- * OO Object declaration
- *
- */
-var OO = {};
-
-/** 
- * OO Object
- *
- * @param {object} options
- * 		required:
- *	 		username - username for OO that is allowed to use the API
- *			password - the password for the username
- *			validSSL - true or false whether to require a valid SSL certificate or allow self-signed certs
- *			baseUrl  - the base url of the REST endpoint for an OO server, for example: http://localhost:8050/oo/rest/v1
- */
-OO.setClient = function(options) {
-
-	if (!options.username || !options.password) {
-		return new Error("no username or password settings provided in options object");
+import 'babel-polyfill';
```

Comment:
Do we really need babe-polyfill for the library itself? Is it required for import? it's a minor thing but I'm not a fan of imports anyway and would resort to just require. Currently we can leave as is though to get the PR pushed without further delay.

About polyfill - what I mean is, there are 2 use cases for it:
1. devs who will be hacking it like you and I, which we can set a guideline for developing it using v7.6 and onwards with async/await etc
2. devs using the library through npm, and for that the webpack config should already do the ES5 conversion when creating the result in `dist/`